exercises/word-search/word-search.js

```diff
@@ -0,0 +1,52 @@
+//
+// This is only a SKELETON file for the 'Word Search' exercise. It's been provided as a
+// convenience to get you started writing code faster.
+//
+
+function searchHorizontally() {
+  throw new Error("Remove this statement and implement this function");
+}
+
+function flipCoordinates() {
+  throw new Error("Remove this statement and implement this function");
+}
+
+function flipGrid() {
+  throw new Error("Remove this statement and implement this function");
+}
+
+function diagonalFind() {
+  throw new Error("Remove this statement and implement this function");
+}
+
+function findAWordDiagonallyTopDown() {
+  throw new Error("Remove this statement and implement this function");
+}
+
+function findAWordDiagonallyBottomUp() {
+  throw new Error("Remove this statement and implement this function");
+}
+
+function formatCoordinates() {
+  throw new Error("Remove this statement and implement this function");
+}
+
+function searchDiagonally() {
+  throw new Error("Remove this statement and implement this function");
+}
+
+function findWordInAnyDirection() {
+  throw new Error("Remove this statement and implement this function");
+}
+
+class WordSearch {
+  constructor() {
+    throw new Error("Remove this statement and implement this function");
+  }
+
+  find() {
+    throw new Error("Remove this statement and implement this function");
+  }
+}
+
+module.exports = WordSearch;
```

Comment:
If you look at the examples and other stubs, you see that we use `es6` `export`, not `CommonJS` `module.exports`.